packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewFlutterPlugin.java

```diff
@@ -50,33 +48,13 @@ public class WebViewFlutterPlugin implements FlutterPlugin, ActivityAware {
    *
    * <p>THIS PLUGIN CODE PATH DEPENDS ON A NEWER VERSION OF FLUTTER THAN THE ONE DEFINED IN THE
    * PUBSPEC.YAML. Text input will fail on some Android devices unless this is used with at least
-   * flutter/flutter@1d4d63ace1f801a022ea9ec737bf8c15395588b9. Use the V1 embedding with {@link
-   * #registerWith} to use this plugin with older Flutter versions.
+   * flutter/flutter@1d4d63ace1f801a022ea9ec737bf8c15395588b9.
```

Comment:
This whole paragraph can go; the hash landed in 2019 and was in Flutter 1.16, so this whole comment has been wrong for quite a long time.